tachyon/math/finite_fields/prime_field_base_unittest.cc

```diff
@@ -110,20 +110,4 @@ TYPED_TEST(PrimeFieldBaseTest, Hash) {
       std::make_tuple(F::Random(), F::Random())));
 }
 
-TYPED_TEST(PrimeFieldBaseTest, JsonValueConverter) {
```

Comment:
Why is this test removed?

Reply:
This is because the serialized text of `PrimeField` is no longer an object.

Reply:
I think it would be nice to still test prime fields' parsing and writing, even though it may seem trivial. Perhaps we can run it like this instead:
```
TYPED_TEST(PrimeFieldBaseTest, JsonValueConverter) {
  using F = TypeParam;

  F expected_point(3);
  std::string expected_json = "\"0x3\"";

  F p;
  std::string error;

  rapidjson::Document document;
  document.Parse(expected_json.data(), expected_json.length());
  base::RapidJsonValueConverter<F>::To(document, "", &p, &error);

  ASSERT_TRUE(error.empty());
  EXPECT_EQ(p, expected_point);

  std::string json = base::WriteToJson(expected_point);
  EXPECT_EQ(json, expected_json);
}